pixart.js

```diff
@@ -1 +1,88 @@
+//when the button is clicked
+var button = document.querySelector("button");
 
+button.addEventListener("click", change);
```

Comment:
Doesn't really matter, but if you're not using this `button` variable again you may as well do everything on one line:

```js
document.querySelector("button").addEventListener("click", change);
```